package com.mv.main.movimentacaos.empresa.exception;

public class ErrorInternal extends RuntimeException{

    public ErrorInternal() {
        super("NÃ£o foi possivel concluir o cadastro, tente novamnete");
    }

    public ErrorInternal(String message) {
        super(message);
    }
}
